src/tasks/leveling.ts

```diff
@@ -110,6 +110,31 @@ const usefulEffects: Effect[] = [
   $effect`Carol of the Hells`,
 ];
 
+const sugarItemsAboutToBreak: () => Item[] = () => {
+  const sugarItems = [
+    { id: 4181, item: $item`sugar chapeau` },
+    { id: 4182, item: $item`sugar shorts` },
+  ];
+  const results = sugarItems
+    .map((entry) => {
+      const { id, item } = entry;
+      const itemAboutToBreak = parseInt(get(`sugarCounter${id.toString()}`), 10) >= 30;
+      print(`Is ${item.name} about to break? ${itemAboutToBreak}`);
+      return itemAboutToBreak ? [item] : [];
+    })
+    .reduce((a, b) => a.concat(b));
+  print(`Sugar items about to break: ${results.map((it) => it.name).join(", ")}`);
+  return results;
+};
+
+const baseOutfit: () => OutfitSpec = () => ({
+  offhand: $item`unbreakable umbrella`,
+  acc1: $item`codpiece`,
+  familiar: $familiar`Cookbookbat`,
+  modifier: "0.25 mys, 0.33 ML, -equip tinsel tights, -equip wad of used tape",
+  avoid: [...sugarItemsAboutToBreak()],
```

Comment:
My end goal is to have the outfit get recalculated as the sugar items are used, so that we can use them when beneficial (e.g. shorts increasing our stat % for powerleveling), but then make sure that once they only have a single turn left, we avoid them to preserve them for the tests.